cmd/tk/bootstrap_github.go

```diff
@@ -417,7 +423,7 @@ func generateGitHubKustomization(url, name, namespace, tmpDir string, interval t
 			Interval: metav1.Duration{
 				Duration: 10 * time.Minute,
 			},
-			Path:  "./",
+			Path:  "./" + targetPath,
```

Comment:
`filepath.Join()`?

Reply:
Doesn't work, as join removes `./` and that breaks the CRD validation.

Reply:
I added a trim to guard against double `./`.